docs/guides/web-servers/nginx/how-to-install-nginx-on-debian/index.md

```diff
@@ -0,0 +1,52 @@
+---
+slug: how-to-install-nginx-on-debian
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: 'Two to three sentences describing your guide.'
```

Comment:
missing og_description, description, and h1_title

Reply:
also external resources,, and tags

Reply:
this was removed